### PR TITLE
Feature/fill software

### PR DESCRIPTION
**Pull request by 0x72D0**
## Description
Add nearly every software node to the project except proc_mapping (currently in refactoring by the others PFE team) and proc_underwater_com (lack maturity).

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by supertoto29 — COMMENTED** [review]
For the moment, there is some node that in local I would remove from the LOCAL.yaml but this can be discussed at a later date

File: src/proc_fault/proc_fault_node.cc [hunk 1]
```diff
@@ -45,25 +52,143 @@ namespace proc_fault
...
     {
-        ros::Rate r(20);
+        ros::Rate r(5);
         while(ros::ok())
         {
             ros::spinOnce();
 
             sonia_common::FaultDetection msg;
             msg.navigation = procFaultModule[0]->checkMonitoring();
+            msg.vision = procFaultModule[1]->checkMonitoring();
```
Comment (supertoto29) [inline src/proc_fault/proc_fault_node.cc:186]: Can you make a loop of it?

**Review by supertoto29 — APPROVED** [review]
Change the detection sequence for a loop if possible. Or else, LGTM!

File: src/proc_fault/proc_fault_node.cc (hunk 1 above)
Comment (0x72D0) [reply src/proc_fault/proc_fault_node.cc:186]: @supertoto29 I don't really understand, it's in the spin it should loop...

**Pull request merged**